src/main/webapp/app/shared/components/copy-icon-button/copy-icon-button.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input } from '@angular/core';
+import { faCopy } from '@fortawesome/free-regular-svg-icons';
+
+@Component({
+    selector: 'jhi-copy-icon-button',
+    templateUrl: './copy-icon-button.component.html',
+})
+export class CopyIconButtonComponent {
+    @Input() onCopyFinished: Function;
```

Comment:
💡A more specific type should be good here as onCopyFinished will be called in this way ```
onCopyFinished($event, onCopyFinishedKey)
```